roles/common/tasks/ssh.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Install openssh-server
+  yum:
+    name: openssh-server
+
+- name: Start ssh service
+  service:
+    name: sshd
+    state: started
```

Comment:
You can also delete this step as the default service is running and enabled

Reply:
Checked and fixed